requirements/runtime.txt

```diff
@@ -1,2 +1,3 @@
 snowballstemmer==1.2.1
 toml==0.10.2
+typing_extensions==4.1.1
```

Comment:
Not sure for what this is used, I don't see it in the pyproject.toml, and also it was already like this before... But ``typing_extensions`` is used by a lot of packages, the constraint should be more forgiving as to be compatible with more of them imo (using >= instead of == at least).

Reply:
This dep is needed for `typing.Literal`, which wasn't introduced until Python 3.8. So this dependency is actually needed. Dependencies in `requirements/*` can and should actually be pinned to a specific version (mostly for CI purposes, we should manage these files with dependabot). For users actually installing pydocstyle, these files aren't used, the deps listed in `pyproject.toml` or `setup.py` are. Before merging, we should rebase this PR on master and add `typing_extensions` to `pyproject.toml`. They don't have a changelog so I don't know what version of typing_extensions added Literal though. Although honestly, we could prob just type things as a string instead of as a Literal. That's probably simpler.